presto/presto.go

```diff
@@ -554,17 +560,23 @@ func (st *driverStmt) QueryContext(ctx context.Context, args []driver.NamedValue
 
 	if len(args) > 0 {
 		hs = make(http.Header)
-		hs.Add(preparedStatementHeader, preparedStatementName+"="+url.QueryEscape(st.query))
-
-		ss := make([]string, len(args))
-		for i, arg := range args {
-			s, err := Serial(arg.Value)
-			if err != nil {
-				return nil, err
+		if args[0].Name == prestoUserHeader {
```

Comment:
why args[0] ?

Reply:
I'm resolving this conversation: we are discussing this in the conversation below.